src/sidebar/components/PendingUpdatesButton.tsx

```diff
@@ -25,10 +26,12 @@ function PendingUpdatesButton({
   onClick,
   toastMessenger,
 }: PendingUpdatesButtonProps) {
+  useShortcut('l', () => hasPendingUpdates && onClick());
```

Comment:
I'm not sure if `l` is the best option. I choose it after "Load", but I guess anything else should be ok.

Reply:
Often what I'll do in these situations (choosing a keyboard shortcut) is visit some large properties: Twitter, GitHub (not relevant in this case), etc. and see how they do it. Or google around a bit to see if there are any common conventions. Also I try to make sure we don't have a conflict with other shortcuts, though this is more of a concern when there are meta-keys involved (e.g. obviously we wouldn't want to assign something to CMD-C). You might just poke around a bit and see if you see any patterns out there.

Reply:
Thanks!

Reply:
I have switched to `.` key. That's what Twitter uses. I haven't been able to find out what Facebook, Instagram or others have (or maybe they don't have, as there's no mention for a shortcut to load posts in their docs)

Reply:
Heh, the challenge with `.` is: how do we announce that? `l` is always `l`, but `.` might be "period" or "full stop" or "dot"...

Reply:
Hm, I didn't think about that.